samples/kotlin-cli/src/main/kotlin/org/jellyfin/sample/cli/command/Ping.kt

```diff
@@ -27,17 +29,17 @@ class Ping(
 
 		val result by api.systemApi.getPublicSystemInfo()
 
-		println("id: ${result.id}")
-		println("name: ${result.serverName}")
-		println("version: ${result.version}")
+		logger.info("id: ${result.id}")
+		logger.info("name: ${result.serverName}")
+		logger.info("version: ${result.version}")
 	}
 
 	private suspend fun runExtended() {
 		val servers = jellyfin.discovery.getRecommendedServers(server)
 		for (server in servers) {
-			println("${server.address}: score=${server.score} duration=${server.responseTime}ms")
-			println("info=${server.systemInfo}")
-			println()
+			logger.info("${server.address}: score=${server.score} duration=${server.responseTime}ms")
+			logger.info("info=${server.systemInfo}")
+			logger.info("")
```

Comment:
When I look at the output, using the logger instead of println() looks better because everything is indented the same.

```
[DefaultDispatcher-worker-2] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for https://demo.jellyfin.org/stable
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for https://demo.jellyfin.org:8920/stable
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for https://demo.jellyfin.org:8096/stable
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET https://demo.jellyfin.org:8920/stable/System/Info/Public
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET https://demo.jellyfin.org:8096/stable/System/Info/Public
[DefaultDispatcher-worker-2] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET https://demo.jellyfin.org/stable/System/Info/Public
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for http://demo.jellyfin.org/stable
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET http://demo.jellyfin.org/stable/System/Info/Public
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for http://demo.jellyfin.org:8096/stable
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET http://demo.jellyfin.org:8096/stable/System/Info/Public
[main] INFO org.jellyfin.sample.cli.command.Ping - https://demo.jellyfin.org/stable: score=OK duration=785ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Success(PublicSystemInfo(localAddress=http://172.17.0.2:8096/stable, serverName=Stable Demo, version=10.8.13, productName=Jellyfin Server, operatingSystem=Linux, id=713dc3fe952b438fa70ed35e4ef0525a, startupWizardCompleted=true))
[main] INFO org.jellyfin.sample.cli.command.Ping - 
[main] INFO org.jellyfin.sample.cli.command.Ping - https://demo.jellyfin.org:8096/stable: score=BAD duration=3530ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out)
[main] INFO org.jellyfin.sample.cli.command.Ping - 
[main] INFO org.jellyfin.sample.cli.command.Ping - https://demo.jellyfin.org:8920/stable: score=BAD duration=3530ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out)
[main] INFO org.jellyfin.sample.cli.command.Ping - 
[main] INFO org.jellyfin.sample.cli.command.Ping - http://demo.jellyfin.org/stable: score=BAD duration=230ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.InvalidStatusException: Invalid HTTP status in response: 302)
[main] INFO org.jellyfin.sample.cli.command.Ping - 
[main] INFO org.jellyfin.sample.cli.command.Ping - http://demo.jellyfin.org:8096/stable: score=BAD duration=3502ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out)
[main] INFO org.jellyfin.sample.cli.command.Ping -
```

Reply:
Hm. Well, is the empty line strictly necessary? Imo it could also be removed.

Reply:
Yeah I would vote for removing the blank lines

Reply:
```
[DefaultDispatcher-worker-2] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for https://demo.jellyfin.org/unstable
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for https://demo.jellyfin.org:8096/unstable
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for https://demo.jellyfin.org:8920/unstable
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET https://demo.jellyfin.org:8096/unstable/System/Info/Public
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET https://demo.jellyfin.org:8920/unstable/System/Info/Public
[DefaultDispatcher-worker-2] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET https://demo.jellyfin.org/unstable/System/Info/Public
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for http://demo.jellyfin.org:8096/unstable
[DefaultDispatcher-worker-5] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET http://demo.jellyfin.org:8096/unstable/System/Info/Public
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.discovery.RecommendedServerDiscovery - Requesting public system info for http://demo.jellyfin.org/unstable
[DefaultDispatcher-worker-4] INFO org.jellyfin.sdk.api.ktor.KtorClient - GET http://demo.jellyfin.org/unstable/System/Info/Public
[main] INFO org.jellyfin.sample.cli.command.Ping - https://demo.jellyfin.org/unstable: score=GREAT duration=866ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Success(PublicSystemInfo(localAddress=http://172.17.0.3:8096/unstable, serverName=Unstable Demo, version=10.9.0, productName=Jellyfin Server, operatingSystem=, id=713dc3fe952b438fa70ed35e4ef0525b, startupWizardCompleted=true))
[main] INFO org.jellyfin.sample.cli.command.Ping - https://demo.jellyfin.org:8096/unstable: score=BAD duration=3543ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out)
[main] INFO org.jellyfin.sample.cli.command.Ping - https://demo.jellyfin.org:8920/unstable: score=BAD duration=3543ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out)
[main] INFO org.jellyfin.sample.cli.command.Ping - http://demo.jellyfin.org/unstable: score=BAD duration=212ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.InvalidStatusException: Invalid HTTP status in response: 302)
[main] INFO org.jellyfin.sample.cli.command.Ping - http://demo.jellyfin.org:8096/unstable: score=BAD duration=3501ms
[main] INFO org.jellyfin.sample.cli.command.Ping - info=Failure(org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out)
```